pkg/compactor/compactor.go

```diff
@@ -60,6 +60,17 @@ var (
 	errInvalidSymbolFlushersConcurrency           = fmt.Errorf("invalid symbols-flushers-concurrency value, must be positive")
 	errInvalidMaxBlockUploadValidationConcurrency = fmt.Errorf("invalid max-block-upload-validation-concurrency value, can't be negative")
 	RingOp                                        = ring.NewOp([]ring.InstanceState{ring.ACTIVE}, nil)
+
+	// compactionIgnoredLabels defines the external labels the compactor
+	// will drop/ignore when planning jobs.
+	// * We remove ingester ID because we don't shard blocks anymore, while still
+	// 	honoring the shard ID if sharding was done in the past.
+	// * We remove tenant ID to make sure that we compact blocks with and without
+	//  the label together.
```

Comment:
Let's format the comment so that it complies with Go doc comments: https://tip.golang.org/doc/comment#lists